include/core/parameters.h

```diff
@@ -740,6 +740,9 @@ namespace Parameters
     /// fraction should be assembled (@p true) or not (@p false).
     bool vof_dcdd_stabilization;
 
+    // Diffusion factor scaling the DCDD stabilization term in the VOF equation
+    double dcdd_diff_coeff;
```

Comment:
```suggestion
    double dcdd_diffusion_coeff;
```
The "diff" abbreviation for me means "difference", although in this context it does not make much sens. (If changed, it will also affect the `.cc`)